js/src/views/Account/account.js

```diff
@@ -80,6 +82,15 @@ class Account extends Component {
     if (prevAddress !== nextAddress) {
       this.setVisibleAccounts(nextProps);
     }
+
+    const { isTestnet } = nextProps;
+    if (typeof isTestnet === 'boolean' && !this.state.verificationStore) {
+      const { api } = this.context;
+      const { address } = nextProps.params;
+      this.setState({
+        verificationStore: new VerificationStore(api, address, isTestnet)
```

Comment:
@ngotchac I put the `verificationStore` into `<Account>` to make it independent from the modal component. Back then, this was the first step of moving away from Redux.

I could wait for `isTestnet` to render the whole account page, but that's out of scope of this PR.